Makefile

```diff
@@ -47,7 +47,7 @@ install: install-user
 
 endif
 
-tests: warn-terraform-version warn-packer-version test-engine validate_configs packer-check
+tests: ghpc install warn-terraform-version warn-packer-version test-engine validate_configs packer-check
```

Comment:
why make install as part of tests? 

Does this mean you now need to be root to be able to make tests? what if the user doesn't wan to make install?  Is this really necessary?

Reply:
also, validate configs already depends on ghpc, so there is no need to add the ghpc dependency here.

Reply:
I wanted to make it visible from anywhere, but it is better to just point to the binary in the repo directly, so that's updated.

You wouldn't need root regardless, `make install` will install locally in that case (~/bin/). Either way, I agree it's better to not mess with the installed version.